frontend/test/metabase/scenarios/README.md

```diff
@@ -6,10 +6,14 @@
 
 ## Metabase QA DB Tests
 
-If you're running Cypress locally and you need to write or run tests for one of the [supported databases](https://github.com/metabase/metabase-qa), please make sure that you have `QA_DB_ENABLED` env for that session only. 
+If you're running Cypress locally and you need to write or run tests for one of the [supported databases](https://github.com/metabase/metabase-qa), make sure you have docker running and start cypress using `yarn-test-cypress-open-qa`.
```

Comment:
```suggestion
If you're running Cypress locally and you need to write or run tests for one of the [supported databases](https://github.com/metabase/metabase-qa), make sure you have docker running and start cypress using `yarn test-cypress-open-qa`.
```

Space instead of dash after yarn in command.